coralproject/talk
# Feat/corl 2548 flatten replies depth

## What does this PR do?
Increases the depth at which replies are automatically flattened to 7.

## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## Does this PR introduce any new environment variables or feature flags? 
No.

## If any indexes were added, were they added to `INDEXES.md`?
n/a

## How do I test this PR?
1. Before checking Create a reply chain > 7 replies long (ideally 8 or 9), see that they are flattened only after the 7th comment.
 
 
## How do we deploy this PR?
No special considerations should be required
